contrib/local.mk.example

```diff
@@ -28,8 +24,24 @@
 #
 # CMAKE_BUILD_TYPE := Debug
 
-# By default, all libraries are statically linked to nvim. Uncomment these
-# entries to use dynamic linking instead.
+# By default, nvim uses bundled versions of its required third-party
+# dependencies.
+# Uncomment these entries to instead use system-wide installations of
+# them.
+#
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_BUSTED=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_DEPS=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_JEMALLOC=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LIBTERMKEY=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LIBUV=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LIBVTERM=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LUAJIT=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_LUAROCKS=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_MSGPACK=OFF
+# DEPS_CMAKE_FLAGS:= -DUSE_BUNDLED_UNIBILIUM=OFF
+
+# By default, all libraries are statically linked to nvim.
+# Uncomment these entries to instead use dynamic linking.
```

Comment:
> this has no effect for non-bundled deps, which are already dynamically linked.

It has also no effect for bundled deps, because they are always statically linked (dynamic libraries are [removed](https://github.com/neovim/neovim/pull/2465/files#diff-c83b027201acb259c4535d64dc4a245eR112)).